document/src/main/kotlin/com/ritense/document/service/JsonSchemaDocumentSpecification.kt

```diff
@@ -57,4 +61,16 @@ class JsonSchemaDocumentSpecification(
             }
         return combinePredicates(criteriaBuilder, predicates)
     }
+
+    companion object {
+        @JvmStatic
+        fun byDocumentDefinitionIdName(name: String): Specification<JsonSchemaDocument> {
+            return Specification { root: Root<JsonSchemaDocument>,
+                                   _: CriteriaQuery<*>?,
+                                   criteriaBuilder: CriteriaBuilder ->
+                // documentRoot.get(DOCUMENT_DEFINITION_ID).get(NAME), documentDefinitionName)
```

Comment:
Please remove the commented code. We have git for that.

Reply:
removed it